internal/engine/dockerprune/docker_pruner.go

```diff
@@ -149,13 +156,52 @@ func (dp *DockerPruner) prune(ctx context.Context, maxAge time.Duration) error {
 		prettyPrintCachePruneReport(cacheReport, l)
 	}
 
-	containerReport, err := dp.dCli.ContainersPrune(ctx, f)
+	return nil
+}
+
+func (dp *DockerPruner) deleteOldImages(ctx context.Context, maxAge time.Duration) (types.ImagesPruneReport, error) {
+	opts := types.ImageListOptions{
+		Filters: filters.NewArgs(
+			filters.Arg("label", docker.BuiltByTiltLabelStr),
+		),
+	}
+	imgs, err := dp.dCli.ImageList(ctx, opts)
 	if err != nil {
-		return err
+		return types.ImagesPruneReport{}, err
 	}
-	prettyPrintContainersPruneReport(containerReport, l)
 
-	return nil
+	toDelete := make(map[string]uint64) // map imageID to size in bytes
```

Comment:
i'm surprised this isn't just a filtered list; were you seeing duplicates in the list that you were trying to avoid?

Reply:
you mean why is it a map instead of a slice? because I needed to keep the image size handy for the report (unless i'm missing an easy way to do that)